index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles/main.css">
+    <title>Document</title>
+</head>
+<body>
+    <div class="container">
+    <header class="portfo-header">
+        <div class="mylogo">My Logo</div>
+        <nav class="menu-button"  onclick="toggle()">
+            <img id="hamburger-btn" src="images/ham-btn.png" alt="menu">
+        </nav>
```

Comment:
- [ ] The logo is on the left and has a link (`a` tag). Please try and fix that.

Reply:
Thank you, I fix it